### PR TITLE
Ports Nekomimetic (felinid language) from /TG/

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request

Felinid language 

## Why It's Good For The Game
It's so bad it's good. And every non human race should have IMO it's own language

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/BeeStation/BeeStation-Hornet/assets/148357670/1ea5baec-87d2-4318-81af-775a3518380c

<details>

## Changelog
:cl: Varo
add: felinid language
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
I approved this PR being made, with no guarantee that it would be merged.

**Comment by DrDuckedGoose** [comment]
Don't take my ``f`` language key!
Truly the felind agenda.

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]
Don't give such name to this language. Give it something better or I will key it.

File: code/modules/language/nekomimetic.dm [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+	desc = "To the casual observer, this langauge is an incomprehensible mess of broken Japanese. To the felinids, it's somehow comprehensible."
+	key = "f"
+	space_chance = 70
+	syllables = list(
+		"neko", "nyan", "mimi", "moe", "mofu", "fuwa", "kyaa", "kawaii", "poka", "munya",
+		"puni", "munyu", "ufufu", "uhuhu", "icha", "doki", "kyun", "kusu", "nya", "nyaa",
+		"desu", "kis", "ama", "chuu", "baka", "hewo", "boop", "gato", "kit", "sune", "yori",
+		"sou", "baka", "chan", "san", "kun", "mahou", "yatta", "suki", "usagi", "domo", "ori",
+		"uwa", "zaazaa", "shiku", "puru", "ira", "heto", "etto"
+	)
```
Comment (EvilDragonfiend) [inline code/modules/language/nekomimetic.dm]: I am fine with felinid saying such weird syllables but the current pool shows too felinid-meme at high possibility. Add more japanese-like syllables to lower the chance of awkward combination.
For example, having more syllables like `"itai", "omae", "akaiku"` bla bla would be more accepting, and I hope that'll be less awkward. Such things shouldn't happen that often. It should come unexpectedly then strike you.

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

**Comment by itsmeow** [comment]
Felinids in their current state are slightly modified humans. Why would they have a culture and language entirely separate from humans. We can assume most felinids are first-generation, that being, they used genetic mutation to get their felinid features rather than being born with it. Genetic research is relatively new.

Adding a unique culture/language to felinids is nonsensical within this context. This is the type of thing that needs hundreds of years to naturally form in the real world, at most, this advanced genetic research has existed for 50-100 years.

**Comment by Tyranicranger4** [comment]
This PR is in conflict with how we've enforced R1 (You Must Roleplay) in a handful cases. Felinid/owo speak - or whatever else you'd call this has, at times, been considered too out-of-character and memey when overused; people have been warned for it, and people have been banned for it. Best case scenario this adds absolutely nothing of value, worst case scenario it's actively detrimental to immersion.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Make the language syllables not annoying/LRP/memey

**Comment by Rukofamicom** [comment]
> Felinids in their current state are slightly modified humans. Why would they have a culture and language entirely separate from humans. We can assume most felinids are first-generation, that being, they used genetic mutation to get their felinid features rather than being born with it. Genetic research is relatively new.

I will use this to block you from saying I can't merge felinids into humans if it's being used to prevent felinids from being split further from humans.

**Comment by Rukofamicom** [comment]
> Make the language syllables not annoying/LRP/memey

I second this, but with more specifics:
Remove every nya and meow from the list.

If it looks like japanese gibberish it's good, if it looks like nyas and meows it's bad.

**Review by Rukofamicom — CHANGES_REQUESTED** [review]
> Make the language syllables not annoying/LRP/memey

I second this, but with more specifics:
Remove every nya and meow from the list.

If it looks like japanese gibberish it's good, if it looks like nyas and meows it's bad.

**Comment by ClownMoff** [comment]
Don't key it! I will change it this afternoon

**Comment by ClownMoff** [comment]
> Adding a unique culture/language to felinids is nonsensical within this context. This is the type of thing that needs hundreds of years to naturally form in the real world, at most, this advanced genetic research has existed for 50-100 years.

Saying a language can't form in 50-100 years is wrong. Also lore reasons for a game with limited lore don't make much sense to me

**Comment by ClownMoff** [comment]
imagen


Every single cringe uwu owo reference has been removed (Except neko because I think it's pretty obvius the word representing felinids whould be in the language). Now. Do we have a problem with stuff like "sensei" or "sakura"?

**Comment by ClownMoff** [comment]
https://cdn.discordapp.com/attachments/565330972707651584/1168862377064804422/Desktop_2023.10.31_-_10.40.34.03.mp4?ex=65534eeb&is=6540d9eb&hm=6794354ab83ccaf1d7214d99256cc4f07d0ce4731cba09db360ccba4a6ebc55c&

**Comment by llol111** [comment]
One of the worst PRs i have seen in a long time
Lets make the species known for being annoying and cringe (rightfully or not) even more cringe and annoying, big brain time

**Comment by ClownMoff** [comment]
Random japanese words arent cringe. It doesnt have the nyaa stuff no more

**Comment by Fronsis** [comment]
> > Random japanese words arent cringe. It doesnt have the nyaa stuff no more
> 
> It absolutley is my dude, get real. Maybe the maintainers will change their minds now that you've removed the worst of the worst, but this is still incredibly cringe and not in line with what i'd like for the roleplay on bee as an admin. Just thinking about all the catpeople puking their japanese gibberish all over common, eugh.

I'm not a fan of cat people in general but.. it's not that different to  when they start spamming moffic, oozeling, IPC, etc the only difference is that you can kinda see the words and have a direct association with japanese, which it's already the lore of catpeople, on the others it just looks strange because it's made up words that are inspired on some languages (i forgot which ones were used for moffic for example) but i'm sure someone native from those countries might go ???? everytime the radio gets spammed with moffic

And even if they do spam it, there's a crime for that and they can be charged with it, i'd rather have them saying random japanese words or made up japanese words than them continue with *licks themselves loudly in the mic.

Or.. just nuke the felinids

**Comment by ClownMoff** [comment]
^^^^^^^^^^^^^^^^

**Comment by EvilDragonfiend** [comment]
> ^^^^^^^^^^^^^^^^

> Or.. just nuke the felinids

this? :trollface:

**Comment by EvilDragonfiend** [comment]
🔑 
Okay, I am sorry to close this with all given change requests, but I really don't think it's going to work in our community.
I would likely accept if the decryption part is more complex than this simple, which means I don't think I will accept this port.

I am really fine with felinids having their own language, but I don't like the original idea of this port.
So, I am denying this because it's a port with cringe contents, not because felinids get their own language.
You'll need to give them properly dedicated language rather than this that's made with cringe intent.

**Comment by ClownMoff** [comment]
https://cdn.discordapp.com/attachments/528596617641852928/928031818052173835/1984.mov

**Pull request closed without merging**